service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -206,6 +218,13 @@ private String getFormattedCareOrderDetails(int numOfChildren,
             isInterim ? " until the end of the proceedings." : ".");
     }
 
+    private String getFormattedInterimSupervisionOrderDetails(int numOfChildren, String caseLocalAuthority) {
```

Comment:
Again there is room to test this in isolation if you feel it's needed.

Reply:
Isn't it tested as part of testing the `getOrderTemplateData` method?